app/controllers/spree/admin/product_catalogs_controller.rb

```diff
@@ -17,10 +18,13 @@ def collection
       end
 
       def load_data
-        @variants = Spree::Product.all.map(&:variants_including_master).flatten
         @product_catalogs = Spree::ProductCatalog.where(store: current_store).order(:name)
       end
 
+      def load_variants
+        @variants ||= Spree::Variant.all
```

Comment:
It looks like this is being used here: https://github.com/geminimvp/solidus_product_feed/pull/11/files#diff-5eaede60dc1db18d7a3cb0345a8d4d29R42